test/test-driver.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+   * @param {!EventEmitter} emitter
+   * @param {string} eventName
+   * @param {number=} eventCount
+   * @return {!Promise<!Object>}
+   */
+module.exports.waitForEvents = function(emitter, eventName, eventCount = 1) {
```

Comment:
let's call this file `utils.js` and merge with `frame-utils.js`.

Reply:
done. Side note: I like the name `driver` better than `utils`  😃

Reply:
Thanks. `driver`, and especially `test-driver`, raises very strong associations in my head with `webdriver` and `chromedriver`

Reply:
This is a long discussion 😄 

We don't like the name `utils`, it reminds us of a place you put stuff that you don't know what to do with. On the other hand, we use drivers to put common test stuff + encapsulate test logic / selectors. This way our tests are much more readable (they are written in "english") + they are not aware of selectors / dom implementation. This way, if we change dom implementation, we will change selectors only in one place (and not in all tests). WDYT?